add tumbleweed support

kshtsk opened the pull request:
not sure it is something tumbleweed specific, but this
what was needed to make it working surprisingly

Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

Commit:
add tumbleweed support

not sure it is something tumbleweed specific, but this
what was needed to make it working surprisingly

Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

jhesketh commented on wasser/__init__.py:
```diff
@@ -395,15 +398,17 @@ def provision_hst(host, status, env):
             os.path.dirname(__file__) + '/snippets/run.cmd',
         ],
-        'into': '/tmp/wasser/bin',
+        'into': f'{wasser_remote_dir}/bin',
```
(nit) should use `os.path.join`

kshtsk committed:
examples/jjb: add tumbleweed job definition

We reuse same config and same workflow since it is mostly the same as Leap
with small differences, for distinguishing the distro there is an image
overrided via environment variable.

Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@gmail.com>
